src/main/java/ca/mcgill/mcb/pcingola/snpEffect/commandLine/SnpEffCmdEff.java

```diff
@@ -629,17 +644,29 @@ else if (outFor.equals("GATK")) {
 						} else if (outFor.equals("TXT")) usage("Output format 'TXT' has been deprecated. Please use 'VCF' instead.\nYou can extract VCF fields to a TXT file using 'SnpSift extractFields' (http://snpeff.sourceforge.net/SnpSift.html#Extract).");
```

Comment:
You removed the option to disable stats? That option should be available, some users want to disable all stats due to performance advantages.